src/extension.ts

```diff
@@ -1219,6 +1220,10 @@ export async function activate(context: vscode.ExtensionContext) {
     });
   });
 
+  registerIDFCommand("espIdf.doctorCommand", async () => {
+    await generateConfigurationReport(context);
```

Comment:
For my system this run very quickly but I still think to put this code inside `vscode.window.withProgress` API with `try/catch` to show error if something went wrong as well